daffodil-core/src/main/scala/edu/illinois/ncsa/daffodil/dsom/TermEncodingMixin.scala

```diff
@@ -114,9 +114,6 @@ trait TermEncodingMixin extends KnownEncodingMixin { self: Term =>
         mg.hasNoSkipRegions &&
           hasTextAlignment
       }
-      case gr: GroupRef => {
```

Comment:
Add scaladoc to this trait to explain what its role is. Maybe rename to something more specific.